Auth0/Authentication.swift

```diff
@@ -33,6 +33,94 @@ public typealias DatabaseUser = (email: String, username: String?, verified: Boo
 public protocol Authentication: Trackable, Loggable {
     var clientId: String { get }
     var url: URL { get }
+    
+    /**
+    Logs in a user using an email and an OTP code received via email (last part of the passwordless login flow)
+
+    ```
+    Auth0
+       .authentication(clientId: clientId, domain: "samples.auth0.com")
+       .login(email: "support@auth0.com", code: "123456")
+       .start { result in
+           switch result {
+           case .success(let credentials):
+               print(credentials)
+           case .failure(let error):
+               print(error)
+           }
+       }
+    ```
+
+    You can also specify audience, scope, and additional parameters
+
+    ```
+    Auth0
+       .authentication(clientId: clientId, domain: "samples.auth0.com")
+       .login(email: "support@auth0.com",
+            code: "123456",
+            audience: "https://myapi.com/api",
+            scope: "openid email",
+            parameters: ["state": "a random state"])
+       .start { print($0) }
+    ```
+
+    When result is `Successful`, a `Credentials` object will be in it's associated value with at least an `access_token` (depending on the scopes used to authenticate)
```

Comment:
```suggestion
    When result is `Successful`, a `Credentials` object will be in its associated value with at least an `access_token` (depending on the scopes used to authenticate)
```

Sentence is a bit awkward, I'm not totally clear what it's trying to say.

Reply:
Fixed in b718c8441f66d8259087aec2166fbac980cef0ce - updated all instances of that sentence.